tensorflow/core/framework/registration/registration.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please don't change copyright year.

Why did you pick 2020? Last I checked we are in 2025.